artefacts/README.md

```diff
@@ -1 +1,16 @@
+[<< Back](https://cntt-n.github.io/CNTT/)
+# Handling of figures
 
+## Table of Contents
+
+* [1 Usage of figures](#1)
+
+<a name="1"></a>
+## 1 Usage of figures
+
+- Figures used in the document should be in Portable Network Graphic (`png`) or in Scalable Vector Graphics (`svg`) format
+- Figures must be stored in the `figures` folder of the document where the figures are used (e.g.: `doc/ref_model/figures/`or `'doc/ref_arch/openstack/figures`)
+- File name of the figures shold contain the Work Stream and the chapter number where the figure is used as a prefix (e.g.: `ra2-ch02-topic-of-the-figure`)
```

Comment:
Are hyphens allowed in file names? I think we have to use underscores, but I maybe wrong.

Reply:
@pgoyal01 we didn't specify wither to use underscore or dash. I think either will do.